packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -528,9 +528,14 @@ class ScrollableState extends State<Scrollable> with TickerProviderStateMixin
   // Returns the offset that should result from applying [event] to the current
   // position, taking min/max scroll extent into account.
   double _targetScrollOffsetForPointerScroll(PointerScrollEvent event) {
-    final double delta = widget.axis == Axis.horizontal
+    double delta = widget.axis == Axis.horizontal
         ? event.scrollDelta.dx
         : event.scrollDelta.dy;
+
+    if (widget.axisDirection == AxisDirection.up || widget.axisDirection == AxisDirection.left) {
```

Comment:
It looks like there's actually already a utility for this used elsewhere in widget code: `axisDirectionIsReversed`. (I just noticed it in the drag code, which is why I didn't mention it originally.)

Reply:
@stuartmorgan used the utility and added test. Let me know if you want me to test horizontal scrollables as well